schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -123,7 +123,8 @@
                 "Trust Failure::Failure to Establish Trust",
                 "Trust Failure::Failure to Verify Content",
                 "Trust Failure::Failure to Verify Receiver",
-                "Trust Failure::Failure to Verify Transmitter"
+                "Trust Failure::Failure to Verify Transmitter",
+				"Privilege Escalation"
```

Comment:
Currently `"Impact"` has the `"gainedPrivileges"` property. By moving this value out of `"LogicalImpact"`, the `"gainedPrivileges"` property no longer makes sense where it is at. Instead, the `"gainedPrivileges"` property should be moved to `"Action"` and should only be allowed for use there when `"ImpactMethod"="Privilege Escalation"`.

@Chris-Turner-NIST  This will involve simplifying the `oneOf` on `"Impact"` and changing `"ImpactMethod"` to be an object type with a similar `oneOf` supporting the  `"gainedPrivileges"` property. I can help you with this.

Reply:
This refactoring is being made in PR #87.